controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -91,7 +91,7 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		instance = &instances.Items[0]
 	case len(instances.Items) > 1:
 		message := fmt.Sprintf("only one instance of DSCInitialization object is allowed. Update existing instance name %s", req.Name)
```

Comment:
Does this message make any sense? req.Name would be the dsci that triggered This event, only possible the 2. dsci but likely. Why does that need to be updated and with what?

Maybe a better error message would be to include all the Instance.Items[*].Names found and suggest something about merging /patching them. The name is semi required to be "default-dsci" as well isn't it?

Reply:
the whole idea is we do not have webhook in place(which should only allow one CR in the cluster) then to have a quick hint to the user "you should not have more than one but which one exactly you want to remove or keep that will be up to the user"
e.g a valid use case:
- install operator
- DSCI CR auto created , with default name "default-dsci"
- user create a new DSCI CR, name it "my-dsci" with a different namespace and turn off monitoring in it
- user can instead delete "default-dsci" and keep "my-dsci" as the one in the cluster

i will say `message := fmt.Sprintf("only one instance of DSCInitialization object is allowed")` is enough

Reply:
That's under the assumption that the user knows anything about those rules. When I encountered the issue in the first place I had only applied 1 dsci my self. To my surprise there was one there already with a different name then what I had previously used.

A huge question is maybe why is there a default one at all? And if there for what ever reason is required to be a default one in place, why isn't that detected and just removed if the user adds their own?